torchci/pages/api/benchmark/list_commits.ts

```diff
@@ -29,7 +29,7 @@ export default async function handler(
 
   // get time series data
   try {
-    const { name, query_params, response_formats, data_type } = params;
+    const { name, query_params, response_formats } = params;
```

Comment:
Same comment, I wonder if we need to use a placeholder here, i.e. `const { name, query_params, response_formats, _ } = params;`

Reply:
This one is fine. This format is basically a fancy way to extract the value of specific keys from the params dictionary, and here we simply aren't extracting the data_type key's value anymore